test/specs/scenarios/proofOfAddress.js

```diff
@@ -251,15 +251,18 @@ export const proofOfAddressScenarios = async (lang = 'en_US') => {
         poaDocumentSelection.clickOnBankIcon()
         poaGuidance.clickOnContinueButton()
         documentUpload.switchToCrossDevice()
-        await takePercySnapshot(driver, 'Continue on your phone screen for PoA')
+        await takePercySnapshot(
+          driver,
+          `Continue on your phone screen for PoA in ${lang}`
+        )
```

Comment:
PoA is only supported for English so it's not necessary to take a screenshot for it in other languages. 

I'm also reasonably sure there is no difference between Cross Device Intro screens, i.e. the "Continue on your phone screen", for PoA vs the regular IDV flows. 🤔

Reply:
Do you think we should consider removing this specific capture totally then?

Reply:
Yeah, I think we should remove it here. I believe this screen's snapshot is already covered in another UI test scenario anyway.